menas/src/main/scala/za/co/absa/enceladus/menas/controllers/SchemaController.scala

```diff
@@ -72,6 +74,19 @@ class SchemaController @Autowired() (
     } yield update
   }
 
+  @GetMapping(path = Array("/export/{name}/{version}"))
+  @ResponseStatus(HttpStatus.OK)
+  def exportSchema(@AuthenticationPrincipal principal: UserDetails,
+                   @PathVariable name: String,
+                   @PathVariable version: Int,
+                   response: HttpServletResponse): CompletableFuture[Array[Byte]] = {
+    attachmentService.getSchemaByNameAndVersion(name, version).map { attachment =>
+      response.addHeader("filename", attachment.filename)
+      response.setContentType(attachment.fileMIMEType)
```

Comment:
This gets overridden by the fact that we are returning `Array[Byte]` and is always set to `application/octet-stream`.
I should write tests for this API endpoint.